var/spack/repos/builtin.mock/packages/depb/package.py

```diff
@@ -18,5 +18,5 @@ class Depb(AutotoolsPackage):
 
     def install(self, spec, prefix):
         # sanity_check_prefix requires something in the install directory
-        # Test requires overriding the one provided by `AutotoolsPackage`
+        # Test requires overriding the one provided by `Package`
```

Comment:
One thing I missed here: `Package` does not define an `install()` at all, so this should be reworded either to remove this line or say something like:

```
# `Package` does not define an `install` method by default.
```